diff --git a/panels_breadcrumbs.module b/panels_breadcrumbs.module
index 320bc4c..0666ac3 100644
--- a/panels_breadcrumbs.module
+++ b/panels_breadcrumbs.module
@@ -266,7 +266,7 @@ function panels_breadcrumbs_page_breadcrumb_content_type_render($subtype, $conf,
   }
 
   // If no breadcrumb configuration was found, fallback to Drupal's default.
-  if (isset($correct_conf) && $correct_conf['panels_breadcrumbs_state']) {
+  if (isset($correct_conf) && !empty($correct_conf['panels_breadcrumbs_state'])) {
     $breadcrumb = panels_breadcrumbs_build_breadcrumb($correct_conf, $contexts);
   }
   else {
