diff --git a/video_filter.module b/video_filter.module
index a0487c8..6acffba 100644
--- a/video_filter.module
+++ b/video_filter.module
@@ -319,7 +319,7 @@ function theme_video_filter_iframe($variables) {
   $video = $variables['video'];
   $classes = video_filter_get_classes($video);
   $attributes = '';
-  if ($video['attributes']) {
+  if (!empty($video['attributes'])) {
     $attributes = drupal_attributes($video['attributes']);
   }
 
